spec/models/transfer/between_plates_spec.rb

```diff
@@ -32,4 +32,39 @@
       expect(destination_well.stock_wells).to eq([source_well])
     end
   end
+
+  context 'with pre-capture pools' do
+    # This test checks the fix that allows creating the BGE Lib PrePool plate
+    # from BGE Lib PCR XP plate when some wells are failed in Limber. Failed
+    # wells are skipped as long as they are not in the transfers specified.
+    let(:plate) do
+      plate = create(:pooling_plate) # 6 wells
+      pools = [%w[A1 B1 C1], %w[D1 E1 F1]] # into 2 pools
+      pools.each_with_index do |locations, index|
+        plate
+          .wells
+          .located_at(locations)
+          .each do |well|
+            # likely to be after ISC submission
+            create(:isc_request, asset: well, pre_capture_pool: pre_capture_pools[index])
+          end
+      end
+      plate
+    end
+    let(:pre_capture_pools) { create_list(:pre_capture_pool, 2) }
+    let(:child) { create(:plate_with_empty_wells) }
+    let(:transfers) do # A1 excluded
```

Comment:
Maybe extend the comment a bit to be clearer. A1 is the well failed in Limber, so is not excluded from the transfers hash passed to SS to pool the wells.

Reply:
I have enhanced the comment to tell what Limber sends and what Sequencescape should do.